### PR TITLE
QUEST_JOBSTEP.tsv // 2188-2241

### PR DESCRIPTION
**Pull request by zaxxer**
next batch of improved wordgroups, articles and quest objective plurals.
some stuff here was already better than in the ealier sections.

also line 2229 and following: Minotaurs might be a single entity or multiple entities of a monster type. I can't be sure.

### CODE REVIEW

**Comment by zaxxer** [comment]
do note i changed all instanced of "the tough leather of Glizardon" to "tough Glizardon leather" because it sounds better in whole sentences. I'm aware that the item name might be called "tough leather of Glizardon" but IMO the context is not lost here and I think that item description might be adjusted accordingly at one point as well.

File: QUEST_JOBSTEP.tsv [hunk 1]
```diff
@@ -2185,60 +2185,60 @@ QUEST_JOBSTEP_20150317_002184	Defeated Necroventer. Go back and report to Chrono
...
+QUEST_JOBSTEP_20150317_002193	You found the lens. Give it to the Quarrel Shooter Master.
 QUEST_JOBSTEP_20150317_002194	Defeat Shadow Gaoler and get %s
-QUEST_JOBSTEP_20150317_002195	Go to Ranger Master in Klaipeda.
-QUEST_JOBSTEP_20150317_002196	Collect Bat Venom Sample
-QUEST_JOBSTEP_20150317_002197	Ranger Master mentioned an assassin named Evonipon and wants the venom sample he used. Collect samples of venoms from the bats in Crystal Mines 1st Floor.
-QUEST_JOBSTEP_20150317_002198	Colleted all the poison samples needed. Take it to Ranger Master.
+QUEST_JOBSTEP_20150317_002195	Go to the Ranger Master in Klaipeda.
+QUEST_JOBSTEP_20150317_002196	Collect Bat Venom Samples
+QUEST_JOBSTEP_20150317_002197	The Ranger Master mentioned an assassin named Evonipon and wants the venom sample he used. Collect samples of venom from the bats in Crystal Mines 1st Floor.
+QUEST_JOBSTEP_20150317_002198	You colleted all the poison samples needed. Take them to the Ranger Master.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: colleted -> collected

File: QUEST_JOBSTEP.tsv [hunk 2]
```diff
@@ -2185,60 +2185,60 @@ QUEST_JOBSTEP_20150317_002184	Defeated Necroventer. Go back and report to Chrono
...
-QUEST_JOBSTEP_20150323_002232	After talking to Elementalist, he disappeared into darkness.{nl}You better go and talk to him.
-QUEST_JOBSTEP_20150323_002233	I told him that it would be obtainable in Sirudgellar Forest.{nl}That's the place where even the famous Rodelero Master's search team ran away from.
-QUEST_JOBSTEP_20150323_002234	I am looking for a trace of a man. His name is Evoniphon.{nl}I heard Sadhu Master at Akmens Ridge has his information so go there.
-QUEST_JOBSTEP_20150323_002235	It's not a huge task. I want you to help my experiment..{nl}First, please bring me burnable juice of Tonturia from Zachariel Intersection.
-QUEST_JOBSTEP_20150323_002236	In fact, Tonturia is nothing, but the fact that there are lots of other monsters nearby is more dangerous.{nl}Hmm...Good Luck!
-QUEST_JOBSTEP_20150323_002237	My father is a merchant who comes to Fedimian often.{nl}But after he went into Sirudgellar Forest do some transaction, he never returned.{nl}
+QUEST_JOBSTEP_20150323_002229	I heard Minotaurs keeps it somewhere in the Sirudgellar Forest.{nl}That monster is not easy so you should be prepared a lot.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Minotaurs -> the Minotaurs (_for a better flow in the sentence_)

**Comment by zaxxer** [comment]
@Senhart: i kept "keeps" as Minotaur is a single entity (boss monster).

General: I removed the ending s from Minotaur and Chapparition as the wiki lists their names like this. https://github.com/Treeofsavior/EnglishTranslation/wiki/2.9.%20Boss%20Monsters

Is this correct?

**Comment by Sunaries** [comment]
Correct

**Comment by imcgames** [comment]
The Pull Request is closed for issues regarding copyrights as stated in our main page. 
We will be updating you with details in the near future regarding this matter so please hold your Pull Requests until things the issue is resolved. 
We apologize for the inconvenience caused.

**Pull request closed without merging**